plugin/evm/vm.go

```diff
@@ -481,6 +484,19 @@ func (vm *VM) Initialize(
 		}
 	}
 
+	// parse and cache valid off-chain warp messages to warp backend
+	for _, message := range vm.config.OffChainWarpMessages {
+		messageByte, err := hex.DecodeString(message)
+		if err != nil {
+			return errInvalidOffChainWarpMessage
+		}
+		unsignedMessage, err := avalancheWarp.ParseUnsignedMessage(messageByte)
+		if err != nil {
+			return errInvalidOffChainWarpMessage
+		}
+		vm.warpBackend.PutMessage(unsignedMessage)
```

Comment:
Should we add an info log here for the node validator to know the message has been added as expected?